CONTRIBUTING.md

```diff
@@ -18,11 +18,14 @@ cd fastplotlib
 pip install -e ".[notebook,docs,tests]
 ```
 
+> If you cloned before installing `git-lfs`, you can run `git lfs pull` at any
+> time to properly download files.
+
 3. Checkout the `main` branch, and then checkout your feature or bug fix branch, and run tests:
 
 > **Warning**
-> Do not commit or add any changes from `examples/screenshots` or `examples/diffs`. 
-> If you are creating new test examples that generate or change screenshots please post an issue on the repo and we will help you. The screenshots will be generated on github actions servers, which you can then copy into the screenshots dir. :) 
+> Do not commit or add any changes from `examples/desktop/screenshots`.
```

Comment:
Keep the diffs part here, if tests produce diffs they will get placed there and these files shouldn't be pushed.

Reply:
I added diffs to git ignore... hence the question in the original post